packages/core/src/index.ts

```diff
@@ -28,10 +31,13 @@ const parseProps = props => {
   return next
 }
 
-export const jsx = (type, props, ...children) =>
+export const jsx: typeof React.createElement = (type, props, ...children) =>
```

Comment:
Although it isn't 100% correct (jsx handles `sx` and `css` props, createElement doesn't), focusing on this now would be bikeshedding.

Reply:
Feel free to start submitting PRs to clean this up :wink: